### PR TITLE
Add Pale Moon version 26.5.0

### PR DESCRIPTION
**Pull request by ibrokemypie**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference]. (.app bundle is new moon, app is pale moon. named cask palemoon)
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

Adds the app Pale Moon (Newmoon.app) for MacOS. Would like to add version 27 seperately as it is a major rebase that breaks some features.

### CODE REVIEW

File: Casks/palemoon.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'palemoon' do
+  version '26.5.0'
```
Comment (miccal) [inline Casks/palemoon.rb]: Please change the `version` to `6.5.0-131`.

File: Casks/palemoon.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  sha256 '617581f55863b2ab0f55538791d2c82a06378febffebce14b8d1c1d625f465cb'
+
+  url 'https://mac.palemoon.org/dist/palemoon-26.5.0-131-gstreamer.en-US.mac64.dmg'
```
Comment (miccal) [inline Casks/palemoon.rb]: Please change this to
```
url "https://mac.palemoon.org/dist/palemoon-#{version}-gstreamer.en-US.mac64.dmg"
```

File: Casks/palemoon.rb (hunk 1 above)
Comment (miccal) [inline Casks/palemoon.rb]: Yes, that is what I meant.

File: Casks/palemoon.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+cask 'palemoon' do
```
Comment (vitorgalvao) [inline Casks/palemoon.rb]: Token reference was not followed.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/palemoon.rb (hunk 3 above)
Comment (ibrokemypie) [inline Casks/palemoon.rb]: token reference says to use newmoon as the file is called NewMoon.app, however the application is official named Pale Moon, so went with that

Comment (vitorgalvao) [inline Casks/palemoon.rb]: And the token reference also explains why that doesn’t matter and why it must be followed.

Comment (ibrokemypie) [inline Casks/palemoon.rb]: have updated to follow token reference

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**